test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2299,7 +2299,7 @@ static public void testCheckCastNullableTypeOnNull() throws Throwable {
 	static public void testValueWithLongAlignmentGCScanning() throws Throwable {
 		ArrayList<Object> longAlignmentArrayList = new ArrayList<Object>(objectGCScanningIterationCount);
 		for (int i = 0; i < objectGCScanningIterationCount; i++) {
-			Object newLongAlignmentArray = Array.newInstance(assortedValueWithLongAlignmentClass, genericArraySize);
+			Object newLongAlignmentArray = (Object)ValueClass.newNullRestrictedArray(assortedValueWithLongAlignmentClass, genericArraySize);
```

Comment:
Aren't the casts to `Object` (here and on the other changed lines) redundant (`newNullRestrictedArray()` is declared to return `Object[]`)?

Reply:
Your right, I will clean this up.